lib/auction/auction.go

```diff
@@ -41,10 +41,9 @@ type Bid struct {
 
 // WinningBid contains details about a winning bid.
 type WinningBid struct {
-	BidderID                peer.ID
-	Acknowledged            bool // Whether or not the bidder acknowledged receipt of the win
```

Comment:
- `Acknowledged`: No longer needed because winners that's don't ack they have won are not selected. This happens in-line during winner selection.
- `ProposalCidAcknowledged`: No longer needed because when winners don't ack the proposal cid delivery, an error is stored in the new field `ErrorCause` and returned to the broker. In this case, the broker would have to decide what to do. Most likely retrying and/or recreating the auction.